packages/sql-mapper/lib/connection-info.js

```diff
@@ -3,13 +3,15 @@
 // (with the exception of SQLite)
 const getConnectionInfo = async (db, connectionString) => {
   let database, host, port, user
+  let dbSystem = 'unknown'
```

Comment:
are those changes needed? Can you add a test for them?

Reply:
Adding a test for the "unknown" case, wait to merge

Reply:
Done.